app/app.go

```diff
@@ -115,15 +113,6 @@ func GetWasmOpts(nibiru NibiruApp, appOpts servertypes.AppOptions) []wasmkeeper.
 		wasmOpts = append(wasmOpts, wasmkeeper.WithVMCacheMetrics(prometheus.DefaultRegisterer))
 	}
 
-	// Add the bindings to the app's set of []wasmkeeper.Option.
-	wasmOpts = append(wasmOpts, wasmbinding.NibiruWasmOptions(
-		nibiru.GRPCQueryRouter(),
-		nibiru.appCodec,
```

Comment:
The stargate queries are still supposed to be there. If that part is removed, contracts can't use the query server at all. 

So the only part to remove here is the keeper executor stuff, not the entire querier